src/main/java/mods/railcraft/common/blocks/tracks/outfitted/kits/TrackKitDumping.java

```diff
@@ -34,7 +34,7 @@ public TrackKits getTrackKitContainer() {
 
     @Override
     public void onMinecartPass(EntityMinecart cart) {
-        if (!isPowered()) {
+        if (!isPowered() && !theWorldAsserted().getBlockState(getPos().down()).getBlock().isVisuallyOpaque()) {
```

Comment:
I'd suggest looking at the block below this as well, since it drops the entity two blocks down.